src/libs/Network.js

```diff
@@ -38,21 +38,20 @@ Onyx.connect({
  * @param {Boolean} request.data.forceNetworkRequest
  * @return {Boolean}
  */
-function shouldMakeRequest(request) {
+function canMakeRequest(request) {
```

Comment:
The doc above says `Checks to see if a request should be made`, should we update that too?